src/cards/top-languages-card.js

```diff
@@ -245,8 +246,23 @@ const useLanguages = (topLangs, hide, langs_count) => {
     .sort((a, b) => b.size - a.size)
     .filter((lang) => {
       return !langsToHide[lowercaseTrim(lang.name)];
-    })
-    .slice(0, langsCount);
+    });
+
+  if (merge_others) {
```

Comment:
@RedHeadphone Thanks for your feature request! Your implementation was, however, not working for me when I set the `lang_count` to less than 10. This was because of this condition

https://github.com/anuraghazra/github-readme-stats/blob/7351c8046440f55e514ddde229c4387272a9bd0e/src/cards/top-languages-card.js#L228

I fixed this, and now the code is correct. There are, however, two things that still need to be done for your PR to be accepted for the Hacktoberfest:
-  [ ] Add some tests for the new `merge_others` argument in the https://github.com/anuraghazra/github-readme-stats/blob/master/tests/renderTopLanguages.test.js test file.
- [ ] Document this new parameter including its default value in [the readme](https://github.com/anuraghazra/github-readme-stats#language-card-exclusive-options).

Reply:
Yeah, I thought others should only appear when crossing 10 languages limit, I might have misread the Issue. Thanks for fixing it and cleaning code as well !

I will start working on those two task

Reply:
@RedHeadphone Ah, I understand!